frontend/lib/src/components/shared/Tooltip/Tooltip.tsx

```diff
@@ -62,13 +62,48 @@ function Tooltip({
   const theme: EmotionTheme = useTheme()
   const { colors, fontSizes, radii, fontWeights } = theme
 
+  const tooltipRef = useRef<HTMLDivElement>(null)
+
   return (
     <StatefulTooltip
+      onOpen={() => {
+        const parentElement = tooltipRef.current?.parentElement
+        if (!parentElement) {
+          return
+        }
+        // if the tooltip is offscreen to the left, move it to the right by the same amount of pixels
+        // use a timeout to that parentElement.getBoundingClientRect returns the correct value; otherwise
+        // I have observed it to be "0".
+        setTimeout(() => {
+          const boundingClientRect = parentElement.getBoundingClientRect()
+          const xCoordinate = boundingClientRect.x
+
+          const overflowRight =
+            xCoordinate + boundingClientRect.width - window.innerWidth
+          const parentsParentElement = parentElement.parentElement
```

Comment:
Maybe add a comment here why we need to get the parentsParent here

Reply:
Added a comment!